package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.51.2",
+  "version": "0.51.3",
```

Comment:
You've got v0.52.0 in the changelog and v0.51.3 here.

Reply:
😱 lol

Reply:
We should probably write a Danger.js check to ensure these match…